shell/packages/sandstorm-ui-autocomplete-input/autocomplete-server.js

```diff
@@ -0,0 +1,16 @@
+Meteor.publish("userContacts", function () {
+  // TODO(someday): make this reactive
```

Comment:
I think this has more than "someday" urgency. As this is written, it looks like a user could still see a 404 on profile pictures that are being concurrently changed by their owner.

The fully reactive join should be similar to, but simpler than, what I implemented [here](https://github.com/sandstorm-io/sandstorm/blob/cc46173039ab6f5e5616ce3b606df7809081a38e/shell/packages/accounts-identity/accounts-identity-server.js#L225-L283).